rust/operator-binary/Cargo.toml

```diff
@@ -1,40 +1,36 @@
 [package]
 authors = ["Lars Francke <lars.francke@stackable.de>"]
 description = "Stackable Operator for druid"
-edition = "2018"
+edition = "2021"
 license = "OSL-3.0"
-name = "stackable-druid-operator-binary"
+name = "stackable-druid-operator"
 repository = "https://github.com/stackabletech/druid-operator"
 version = "0.2.0-nightly"
 build = "build.rs"
 
 [dependencies]
```

Comment:
In my opinion, the patch version of the dependencies is not useful in the `Cargo.toml`.

There are also some outdated versions listed.

Reply:
good point, will change to minor versions.